docs/_layouts/default.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <title>Expensify Help</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no, minimal-ui">
-    <link rel="icon" type="image/png" href="https://d2k5nsl2zxldvw.cloudfront.net/images/expensify-favicon.png">
+    <link rel="icon" type="image/png" href="../assets/images/expensify-logo-round.png">
```

Comment:
Using this path fixes the console error
`href="/assets/images/expensify-logo-round.png"`

Reply:
we talked 1:1 on this one and got it squared away 👍